src/openrct2/ride/TrackDesignSave.cpp

```diff
@@ -446,11 +443,7 @@ static void track_design_save_pop_tile_element_desc(const ObjectEntryDescriptor&
     for (size_t i = 0; i < _trackSavedTileElementsDesc.size(); i++)
     {
         TrackDesignSceneryElement* item = &_trackSavedTileElementsDesc[i];
-        if (item->x != tileLoc.x)
```

Comment:
You also need to remove the `tileLoc` variable entirely (it causes the CI to fail).